src/solvers/dgsem_tree/dg_2d_subcell_limiters.jl

```diff
@@ -470,6 +470,9 @@ end
 For subcell limiting, the calculation of local bounds for non-periodic domains require the boundary
 outer state. This function returns the boundary value at time `t` and for node with spatial
 indices `indices`.
+
+!!! warning "Experimental implementation"
+    This is an experimental feature and may change in future releases.
```

Comment:
I forgot to add this warning in the last commit :scream: I hope it is okay if I add it now.